protocol/x/sending/keeper/msg_server_create_transfer_test.go

```diff
@@ -63,14 +63,14 @@ func setUpTestCase(
 ) {
 	// Initialize Mocks and Context.
 	mockKeeper = &mocks.SendingKeeper{}
-	ctx, _, _, _, _, _, _, _, _ := keepertest.SendingKeepers(t)
-	ctx = ctx.WithBlockHeight(25)
+	ks := keepertest.SendingKeepers(t)
+	ks.Ctx = ks.Ctx.WithBlockHeight(25)
```

Comment:
Inline:

```
tc.setupMocks(ks.Ctx.WithBlockHeight(25), mockKeeper)
```
Note `WithBlockHeight` returns a copy and doesn't modify the original context. `ks.Ctx = ks.Ctx.WithBlockHeight(25)` is actively modifying `ks.Ctx`

Reply:
Yeah I think in this case we want to return the modified context at the end of this function. However, I can change it to `ctx := ks.Ctx.WithBlockHeight(25)` and return `ctx` at the end.